### PR TITLE
Search up the tree for the config file.

### LINKED ISSUE #4: Search `.shipit` file among folder ancestors

### PR DESCRIPTION
**Pull request by Asuza**
Fixes #4.

### CODE REVIEW

File: bin/shipit [hunk 1]
```diff
@@ -13,7 +13,9 @@
 
 
 VERSION="0.1.0"
-CONFIG=./.shipit
+CONFIG_DIR=`pwd`
+CONFIG_NAME=.shipit
+CONFIG="$CONFIG_DIR$CONFIG_NAME"
```
Comment (sapegin) [inline bin/shipit]: Looks like you’re overwriting this value anyway, could be empty by default.

Comment (Asuza) [reply bin/shipit]: It's there so if the config file is never found, the existing error message still shows up as expected. I'm pretty sure that conditional wasn't working when I left this blank.

The value only gets overwritten if the config exists somewhere in the tree.

Comment (sapegin) [reply bin/shipit]: I believe it will work like this: `test -f "$CONFIG"`, with quotes.

Comment (sapegin) [reply bin/shipit]: I believe it will work like this: test -f "$CONFIG", with quotes.

**Comment by Asuza** [comment]
@sapegin, I updated this PR as requested.

**Comment by Asuza** [comment]
Do I need to bump the version after my two PRs are merged? Or do you need to do that?

**Comment by sapegin** [comment]
Thanks! I’ll update the version myself.

**Comment by sapegin** [comment]
Out in [0.2.0](https://github.com/sapegin/shipit/releases/tag/0.2.0).

**Pull request merged**